aten/src/ATen/native/native_functions.yaml

```diff
@@ -9222,34 +9222,67 @@
   dispatch:
     CompositeExplicitAutograd: upsample_nearest1d
 
+- func: upsample_nearest_exact1d.vec(Tensor input, int[]? output_size, float[]? scale_factors) -> Tensor
+  python_module: nn
+  dispatch:
+    CompositeExplicitAutograd: upsample_nearest_exact1d
```

Comment:
Perhaps it'd be better to add a flag to `upsample_nearest_*d` instead of adding all these new ops?

Either way, we'll have to wait the FC window of 2 weeks before using the changes, whether the result is new ops or a flagged version of the existing ops.